src/Utilities/RunTimeManager.cpp

```diff
@@ -20,6 +20,9 @@
 
 namespace qmcplusplus
 {
+template class RunTimeManager<CPUClock>;
+template class RunTimeManager<FakeCPUClock>;
+
 RunTimeManager<CPUClock> run_time_manager;
 
 template class RunTimeManager<CPUClock>;
```

Comment:
explicit template definition are repeated now. Please clean it up. Better before run_time_manager instead of after? Not fully sure which is better.

Reply:
ugh.  Based on the errors I got linking before.  I don't think the one for RunTimeManager<CPUClock> should even be necessary with the global, but it is at least with clang8 + gcc ld.